atlp-rwanda/stackup2-barefoot-backend
# #171380739 setup dotenv

#### What does this PR do?
Setup dotenv as a way to load environment variables from .env file into process.env
#### Description of Task to be completed?
- add more environment variables to .env.example
- Install `dotenv` npm package to load .env file KEY=VALUE content into process.env
#### How should this be manually tested?
- After cloning the project repo, run `npm i` inside the cloned repo folder
- run the following commands respectively:
`git fetch`
`git checkout ch-setup-dotenv-171380739`
`npm run start`
- If the above steps have not been followed correctly, the app should throw an error
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#171380739
#### Screenshots (if appropriate)
N/A
